### PR TITLE
Let payment channels work

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +11,12 @@
...
     "debug": "^3.1.0",
+    "express": "^4.16.3",
     "ilp-plugin-mini-accounts": "^3.1.0",
     "ilp-store-memory": "^1.0.0",
-    "machinomy": "^1.6.1",
-    "web3": "0.20.2"
+    "machinomy": "1.8.1"
+  },
+  "devDependencies": {
+    "truffle-hdwallet-provider": "https://github.com/machinomy/truffle-hdwallet-provider"
```
Comment (sharafian) [inline package.json]: Can we pin this to a specific commit, to prevent breaking updates in the future?

Comment (sharafian) [reply package.json]: Or you could publish it under your own NPM namespace and then use that here; Git dependencies can be buggy in the latest NPM version

File: test-infura-sell.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    account: account,
+    db: 'nedb://./server',
+    port: 6666,
+    _store: new Store(),
+    provider: provider,
+    minimumChannelAmount: '10000',
+    debugHostIldcpInfo: {
+        clientAddress: 'test.example',
+        assetCode: 'ETH',
+        assetScale: 6
```
Comment (sharafian) [inline test-infura-sell.js]: The scale of Wei should be 18. It's a little crazy so we may want to update it in the future to use something like MWei or GWei.

File: test.js [hunk 3]
```diff
@@ -6,7 +6,9 @@ const plugin = new Plugin({
   port: 6666,
   _store: new Store(),
   debugHostIldcpInfo: {
-    clientAddress: 'test.example'
+    clientAddress: 'test.example',
+    assetCode: 'ETH',
+    assetScale: 6
```
Comment (sharafian) [inline test.js]: see above comment on scale

**Review by sharafian — CHANGES_REQUESTED** [review]
Just a couple of small things, otherwise looks good

**Review by sharafian — APPROVED** [review]

**Comment by ukstv** [comment]
Fixed.

**Comment by ukstv** [comment]
Yo buddy, is it good to be merged? @sharafian

**Comment by sharafian** [comment]
Tried to merge from my mobile browser but it says there were conflicts. I'm flying in a couple hours but then I'll take a look

**Comment by ukstv** [comment]
This branch has no conflicts with the base branch, says GitHub ¯ \ _ (ツ) _ / ¯

**Pull request merged**